src/include/platform/internal/GenericConfigurationManagerImpl.h

```diff
@@ -77,6 +77,13 @@ class GenericConfigurationManagerImpl
     CHIP_ERROR _GetDeviceCertificate(uint8_t * buf, size_t bufSize, size_t & certLen);
     CHIP_ERROR _GetDeviceIntermediateCACerts(uint8_t * buf, size_t bufSize, size_t & certsLen);
     CHIP_ERROR _GetDevicePrivateKey(uint8_t * buf, size_t bufSize, size_t & keyLen);
+    CHIP_ERROR _GetDeviceDACCertificate(uint8_t * buf, size_t bufSize, size_t & dacLen);
+    CHIP_ERROR _GetDevicePAICertificate(uint8_t * buf, size_t bufSize, size_t & paiLen);
+    CHIP_ERROR _GetDevicePAACertificate(const uint8_t * skid, size_t skidLen, uint8_t * buf, size_t bufSize, size_t & paaLen);
+    CHIP_ERROR _GetDeviceCertificationDeclaration(uint8_t * buf, size_t bufSize, size_t & certDeclLen);
+    CHIP_ERROR _GetDeviceAttestationSignature(const uint8_t * attestationElements, size_t attestationElementsSize,
+                                              const uint8_t * attestationChallenge, size_t attestationChallengeSize, uint8_t * buf,
+                                              size_t bufSize, size_t & signatureLen);
```

Comment:
Having these default implementations of specific certificates here will make it harder to get to the real data per platform, and will also cost flash for the PAA on target devices. The API for accessing PAAs should not be baked here, as it is implementation-specific per commissioner. Lookup of PAA may not even be needed if attestation if verified through a cloud or out-of-SDK app logic. Suggest removing PAA getter from here.